product_docs/docs/net_connector/8.0.5.1/02_requirements_overview.mdx

```diff
@@ -2,11 +2,9 @@
 title: "Product compatibility"
 ---
 
-<div id="requirements_overview" class="registered_link"></div>
+The following sections detail the supported platforms and database versions for the EDB .NET Connector.
 
-These are the supported versions and platforms for the EDB .NET Connector.
-
-The EDB .NET Connector is certified with EDB Postgres Advanced Server version 11 and later.
+## Supported platforms
```

Comment:
PENDING: Replace the windows information with a blanket statement that says .NET runs on all environments where EPAS runs? 

Or does it only work on Windows so this info can remain the same?